.storybook/assets/global.css

```diff
@@ -5,8 +5,9 @@
 body {
     color: var(--color-text-primary);
     background: var(--color-bg-primary);
+    padding: 0 !important;
```

Comment:
Do not use `!important`. wtf, use nesting if you want to have more priority on the selector

Reply:
It's `global.css` so forget my words